html/pfappserver/lib/pfappserver/Form/Config/Authentication/Source/Eduroam.pm

```diff
@@ -0,0 +1,95 @@
+package pfappserver::Form::Config::Authentication::Source::Eduroam;
+
+=head1 NAME
+
+pfappserver::Form::Config::Authentication::Source::Eduroam
+
+=cut
+
+=head1 DESCRIPTION
+
+Form definition to create or update an Eduroam authentication source.
+
+=cut
+
+use strict;
+use warnings;
+
+use HTML::FormHandler::Moose;
+
+extends 'pfappserver::Form::Config::Authentication::Source';
+with 'pfappserver::Base::Form::Role::Help';
+
+has_field 'tlrs1_server_address' => (
```

Comment:
Hmm TLRS stands for Top-Level RADIUS Server.
Don't think it's a "North-America" thing.